internal/namespaces/config/commands.go

```diff
@@ -749,3 +787,79 @@ func getProfile(config *scw.Config, profileName string) (*scw.Profile, error) {
 	}
 	return profile, nil
 }
+
+func validateProfile(profile *scw.Profile) error {
+	if profile.AccessKey != nil {
```

Comment:
Could you extract functions that perform all those operations? Maybe it could be helpful to reuse them at different place in the CLI and SDK?